Umweltzone/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,185 @@
+    <string name="city_info_zone_number_since_text_fragment_yellow">jaune ou verte</string>
+    <string name="city_info_zone_number_since_text_fragment_red">rouge, verte ou jaune</string>
+    <string name="city_info_zone_number_none">
+        Tous les véhicules sont autorisés à circuler dans la zone
+        à faibles émissions, quelque soit leur niveau de pollution.</string>
+    <string name="city_info_next_zone_number_as_of">
+        "Depuis le <xliff:g example="Feb 1, 2016" id="date">%1$s</xliff:g> seuls
+        les véhicules avec un autocollant de couleur <xliff:g example="with a green" id="with_colors">%2$s</xliff:g>
+        sont autorisés à circuler dans les zones à faibles émissions."</string>
+    <string name="city_info_abroad_licensed_vehicle_zone_info">
+        "Les véhicules immatriculés à l&#8217;étranger sont autorisés à circuler dans la zone à faibles émissions
+        jusqu&#8217;au <xliff:g example="Jan 31, 2016" id="date">%1$s</xliff:g> avec un autocollant de couleur
+        <xliff:g example="with a yellow or a green" id="with_colors">%2$s</xliff:g>"</string>
+    <string name="city_info_list_of_cities">
+        "La zone comprend les villes et les régions suivantes :
+        <xliff:g example="Bochum, Castrop-Rauxel, Essen" id="cities">%1$s</xliff:g> et
+        <xliff:g example="Recklinghausen" id="last_city">%2$s</xliff:g>.     "</string>
+    <string name="city_info_further_information">Plus d&#8217;informations en ligne</string>
+    <string name="city_info_badge_online_title">Formulaire de demande d&#8217;autocollant en ligne</string>
+    <string name="city_info_geometry_source_text">"Source des coordonnées satellite:
+        <xliff:g example="Geoportal Berlin / Umweltzone Berlin" id="geometrySource">%1$s</xliff:g>"</string>
+    <string name="city_info_geometry_updated_at_date_format">dd MM YYYY</string>
+    <string name="city_info_geometry_updated_at_text">Dernière mise à jour du tracé de la zone : <xliff:g example="Sep 6, 2014" id="updatedAt">%1$s</xliff:g></string>
+    <string name="city_info_allowed_exhaust_emission_standard_in_diesel_prohibition_zone">
+        "Seuls les véhicules qui respectent les standard d&#8217;émissions Euro <xliff:g example="6" id="number">%1$d</xliff:g>
+        sont autorisés à circuler dans la zone d&#8217;interdiction des diesels."</string>
+    <string name="city_info_zone_number_for_residents_since">
+        "Pour les résidents, ces règles s&#8217;appliquent à partir du <xliff:g example="May 31, 2018" id="date">%1$s</xliff:g>."</string>
+    <string name="city_info_zone_number_for_non_residents_since">
+        "Pour les non-résidents, ces règles s&#8217;appliquent à partir du <xliff:g example="May 31, 2018" id="date">%1$s</xliff:g>."</string>
+    <string name="city_info_provision_applies_to_vehicles">"        Ces règles s&#8217;appliquent aux <xliff:g example="cars and heavy goods vehicles" id="vehicles">%1$s</xliff:g>.     "</string>
```

Comment:
Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.